ckan/lib/redis.py

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+
+u'''
+Redis utilities.
+
+.. versionadded:: 2.6
+'''
+
+from __future__ import absolute_import
+
+import datetime
+import logging
+
+from pylons import config
+from redis import ConnectionPool, Redis
+
+
+log = logging.getLogger(__name__)
+
+REDIS_URL_SETTING_NAME = u'ckan.redis.url'
+
+REDIS_URL_DEFAULT_VALUE = u'redis://localhost:6379/0'
```

Comment:
> Internally, RQ queue names are prefixed with a string containing the CKAN site ID to avoid key collisions when the name Redis database is used for multiple CKAN instances.

So if understand correctly we only might run into problems if we have two CKAN instances sharing a Redis database without having changed their site ids. I think that's an unlikely scenario (if you have multiple CKANs sharing resources with the same site id you are going to get into trouble anyway, starting with Solr) and that a hard failure will add an extra step to deployment for people that want to get up and running fast, so I'd be happy with the current default option.